### PR TITLE
Feature/observe personal information

### PR DESCRIPTION
**Pull request by Jana211**

### CODE REVIEW

File: PatientWebApplication/HealthClinic.DAL/Repository/IPatientsRepository.cs [hunk 1]
```diff
@@ -9,5 +9,7 @@ public interface IPatientsRepository
         PatientUser Find(int id);
         PatientUser Validate(PatientUser patient);
         List<PatientUser> GetAll();
+
```
Comment (trki293) [inline PatientWebApplication/HealthClinic.DAL/Repository/IPatientsRepository.cs:12]: Unnecessarily added blank line, it is necessary to delete.

File: PatientWebApplication/HealthClinic.DAL/Service/IPatientService.cs [hunk 2]
```diff
@@ -15,6 +15,8 @@ public interface IPatientService
         PatientUser Create(PatientDto patientDto);
         List<PatientUser> GetAll();
 
-        string ImageToSave(FileModel file);
+        string ImageToSave(string path, FileModel file);
+
```
Comment (trki293) [inline PatientWebApplication/HealthClinic.DAL/Service/IPatientService.cs:19]: Unnecessarily added blank line, it is necessary to delete.

File: PatientWebApplication/HealthClinic.DAL/Service/PatientService.cs [hunk 3]
```diff
@@ -54,18 +54,16 @@ public List<PatientUser> GetAll()
...
+        public string ImageToSave(string path, FileModel file)
         {
             try
             {
-                string path = Directory.GetParent(Directory.GetCurrentDirectory()).FullName + "\\HealthClinic.DAL\\wwwroot\\" + file.FileName;
-
-
-                using (Stream stream = new FileStream(path, FileMode.Create))
+                string fullPath = path + "\\images\\" + file.FileName;
+                using (Stream stream = new FileStream(fullPath, FileMode.Create))
```
Comment (trki293) [inline PatientWebApplication/HealthClinic.DAL/Service/PatientService.cs:62]: The fullPath variable needs to be deleted and defined in the FileStream constructor.

**Review by trki293 — CHANGES_REQUESTED** [review]
Necessary changes to the code in the indicated places.

**Review by trki293 — APPROVED** [review]
All Clean code principles are met, therefore PR is approved.

**Review by Nemanja0505 — APPROVED** [review]
PR is approved, all Clean code principles are followed.

**Pull request merged**